Core1Platform/BasicPhilosophy/03_InnovativeArchitecture.md

```diff
@@ -0,0 +1,24 @@
+# Innovative MACH+ architecture
+
+The MACH+ approach is considered as fundamental for the newest generation of software and system integrators to quickly adapt new evolutions, trends, and technologies. It is an industry standard that describes modern technology: Microservices based, API-first, cloud-native SaaS (software-as-a-service) and headless. Our MACH+ certification gives enterprises confidence that they are choosing best-in-class vendors that can deliver future-proof technology.
```

Comment:
: microservices

(When a colon introduces a list of things, do not capitalize the first word after the colon unless it is a proper noun.)
https://www.grammarly.com/blog/capitalization-after-colons/

Reply:
d